apps/desktop-wallet/src/modals/AddressOptionsModal.tsx

```diff
@@ -137,6 +166,21 @@ const AddressOptionsModal = memo(({ id, addressHash }: ModalBaseProp & AddressOp
       />
       <HorizontalDivider narrow />
       <ModalFooterButtons>
+        {address.isDefault ? (
+          <div
+            data-tooltip-id="default"
+            data-tooltip-content={t('To delete this address set another one as the default one first.')}
+            style={{ width: '100%' }}
+          >
+            <ModalFooterButton role="secondary" variant="alert" disabled={true}>
+              {t('Delete')}
+            </ModalFooterButton>
+          </div>
+        ) : (
+          <ModalFooterButton role="secondary" variant="alert" onClick={handleDeletePress} disabled={address.isDefault}>
+            {t('Delete')}
```

Comment:
I wanted to suggest to use another word, but I remembered that you used "Forget" in your follow up PR, which I like :)